source/elements/oneDAL/source/terminology.rst

```diff
@@ -6,6 +6,9 @@ Terminology
 
 .. glossary::
 
+   API 
+     Application Program Interface
```

Comment:
Isn't it Application Programming Interface?

Reply:
yes. Interesting that i've take this from one of our documents.. 
Fixed

Reply:
If it's the one I own, please send me the link.